### ISSUE TITLE
Need more information about self sever for debugging or solving some small problems

### ISSUE THREAD

**Issue by BloodSpiritLu** [StunlockStudios/vrising-dedicated-server-instructions#43]
Hi, Firstly, this is a great game. My friends and I, we all like it very much. But due to the connecting issue, we creat our own sever.

**Comment by BloodSpiritLu** [comment]
Thi is the first time for us to creat a serve, So, could u please offer some document about the parameter refer to the commands, introduction of the debug process or other things like this for some new player with the severs? Thanks

**Comment by BloodSpiritLu** [comment]
Oh, there is an other issue here. We have open the port 9876 and query port 9877 in protocol UDP. But, some new palyers still cant search it on server list and can only connect it with the IP. I really want to know how to debug and solve this problem if there is any document offered

**Comment by fishd72** [comment]
I'd imagine that the documentation will improve in time, however on your last point about servers not appearing in the Steam server list... this issue is not specific to V Rising, we had this issue also with Valheim and I see posts all over the internet about this problem with various Steam based games. I'm not sure there will be much Stunlock can do about this.

**Comment by BloodSpiritLu** [comment]
@fishd72 Thanks for your repeating. Its so sad to know the server port problem is related to steam sever, steam should do something to solve it due to so many games selling on it.
Looking forward to your document about console and other sever setting.

**Comment by BloodSpiritLu** [comment]
BTW, I have some suggesting that if you can offer us (author of each sever) more authorities to set our sever like destroyed buildings can recover after the war time, or just randomly drop items when storeage be destoried. I think more freedom to set the sever, more players in different situation can join different sever to realize their dreams.

**Comment by lucsdo** [comment]
> Ah, tem outra questão aqui. Abrimos a porta 9876 e consultamos a porta 9877 no protocolo UDP. Mas, alguns novos jogadores ainda não podem pesquisá-lo na lista de servidores e só podem conectá-lo com o IP. Eu realmente quero saber como depurar e resolver esse problema se houver algum documento oferecido

are you using port 9876 - 9877?

**Comment by BloodSpiritLu** [comment]
> > Ah, tem outra questão aqui. Abrimos a porta 9876 e consultamos a porta 9877 no protocolo UDP. Mas, alguns novos jogadores ainda não podem pesquisá-lo na lista de servidores e só podem conectá-lo com o IP. Eu realmente quero saber como depurar e resolver esse problema se houver algum documento oferecido
> 
> are you using port 9876 - 9877?

YES, After latest update. I changed portto 20xxx, but it always tell officialservicelist failed and connect to port 443 overtime
